src/methods/SANA.cpp

```diff
@@ -2839,12 +2848,13 @@ void SANA::hillClimbingIterations(long long int iterTarget) {
     long long int iter = 1;
 
     initDataStructures(startA);
-    Temperature = 0;
+    Temperature = TInitial;
```

Comment:
This is _definitely_ wrong. Hill climbing is *supposed* to happen at temperature zero.